docs/docs/running-hello-world.md

```diff
@@ -225,10 +225,11 @@ To generate `host-release.jar`:
     On a Linux machine that meets the [system requirements](enclave-modes.md#system-requirements), you can run your
     app in all the modes the same way as in mock mode.
 
-=== "macOS"
+=== "macOS/Windows"
```

Comment:
Please have a look at the docs changes @JohnJoser3

Reply:
Can you point me to those changes? Does not master have the latest doc changes?

Reply:
No, that was a comment to John!